cheroot/test/test_core.py

```diff
@@ -81,7 +82,6 @@ def testing_server(wsgi_server_client):
     """Attach a WSGI app to the given server and preconfigure it."""
     wsgi_server = wsgi_server_client.server_instance
     wsgi_server.wsgi_app = HelloController()
-    wsgi_server.max_request_body_size = 30000000
```

Comment:
Rather than changing a fixture that a lot of tests use, please create a separate fixture if your test needs a different config.

Reply:
My reasoning here is that max_request_body_size is 0 by default, therefore the tests that use this fixture will not be covering a typical case. Furthermore, it doesn't seem like any of the tests in test_core deal with max_request_body_size.

(I'm happy to create a new fixture though.)

Reply:
I think at least one of those tests needs it. But the point is that unrelated changes should be submitted via separate PRs.